exporter/signalfxexporter/translation/constants.go

```diff
@@ -48,25 +53,44 @@ translation_rules:
 
     # kubeletstats receiver metrics
     container.cpu.time: container_cpu_utilization
-    container.memory.usage: container_memory_usage_bytes
-    container.filesystem.usage: container_fs_usage_bytes
     container.filesystem.available: container_fs_available_bytes
     container.filesystem.capacity: container_fs_capacity_bytes
+    container.filesystem.usage: container_fs_usage_bytes
+    container.memory.usage: container_memory_usage_bytes
+    k8s.node.cpu.utilization: cpu.utilization
 
     # k8s cluster receiver metrics
-    k8s/pod/phase: kubernetes.pod_phase
-    k8s/deployment/available: kubernetes.deployment.available
-    k8s/deployment/desired: kubernetes.deployment.desired
     k8s/container/cpu/limit: kubernetes.container_cpu_limit
     k8s/container/cpu/request: kubernetes.container_cpu_request
     k8s/container/memory/limit: kubernetes.container_memory_limit
     k8s/container/memory/request: kubernetes.container_memory_request
     k8s/container/ready: kubernetes.container_ready
-    k8s/node/condition_out_of_disk: kubernetes.node_out_of_disk
-    k8s/node/condition_ready: kubernetes.node_ready
-    k8s/node/condition_p_i_d_pressure: kubernetes.node_p_i_d_pressure
+    k8s/container/restarts: kubernetes.container_restart_count
+    k8s/cronjob/active_jobs: kubernetes.cronjob.active
+    k8s/daemon_set/current_scheduled_nodes: kubernetes.daemon_set.current_scheduled
+    k8s/daemon_set/desired_scheduled_nodes: kubernetes.daemon_set.desired_scheduled
+    k8s/daemon_set/misscheduled_nodes: kubernetes.daemon_set.misscheduled
+    k8s/daemon_set/ready_nodes: kubernetes.daemon_set.ready
+    k8s/deployment/available: kubernetes.deployment.available
+    k8s/deployment/desired: kubernetes.deployment.desired
+    k8s/job/active_pods: kubernetes.job.active
+    k8s/job/desired_successful_pods: kubernetes.job.completions
+    k8s/job/failed_pods: kubernetes.job.failed
+    k8s/job/max_parallel_pods: kubernetes.job.parallelism
+    k8s/job/successful_pods: kubernetes.job.succeeded
+    k8s/namespace/phase: kubernetes.namespace_phase
     k8s/node/condition_memory_pressure: kubernetes.node_memory_pressure
     k8s/node/condition_network_unavailable: kubernetes.node_network_unavailable
+    k8s/node/condition_out_of_disk: kubernetes.node_out_of_disk
+    k8s/node/condition_p_i_d_pressure: kubernetes.node_p_i_d_pressure
```

Comment:
Weird, is `k8s/node/condition_p_i_d_pressure` getting emitted? Should probably be `k8s/node/condition_pid_pressure`  @asuresh4

Reply:
I'm going to handle this as the next PR with updating k8s cluster receiver to otel convention

Reply:
This is likely an artifact of the helper that converts camelcase to snakecase.